lists/bo/bo-cins

```diff
@@ -0,0 +1,40 @@
+{
+  "name": {
+    "en": "Bolivia Institutional Classifier",
+    "local": "Bolivia Clasificador Institucional"
+  },
+  "url": "https://www.sicoes.gob.bo/portal/clasificadores/institucional.php",
+  "description": {
+    "en": "The institutional classifier reflects the organizational structure of the public sector, taking into account the legal nature, institutional level and entity (code, name and acronym), for the administration of State resources, according to competencies in current regulations."
+  },
+  "coverage": [
+    "BO"
+  ],
+  "structure": [
+    "government_agency"
+  ],
+  "sector": [],
+  "code": "BO-CINS",
+  "confirmed": true,
+  "listType": "primary",
```

Comment:
Are we sure that this is a primary list? It would be quite unusual for the contracting system to be issuing these identifiers in my experience. I think it is more likely that the list of public entities is maintained by a central government department or ministry and that SICOES is reusing that list.

Reply:
Cheers Simon I've changed to "secondary" now